rotkehlchen/tests/api/test_nfts.py

```diff
@@ -948,3 +949,84 @@ def mock_uniswap_v3_balances(*_args, **_kwargs) -> dict[str, Any]:
     ))
     result = assert_proper_response_with_result(response)
     assert len(result) == 4
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4, TEST_ACC5]])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('ethereum_modules', [['nfts']])
+def test_customized_queried_addresses(rotkehlchen_api_server):
+    """
+    Test that if queried addresses are customized for nfts module, then from /nfts/balances only
+    NFTs of those addresses are returned"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+
+    def mock_get_all_nft_data(_addresses, **_kwargs) -> tuple[dict[str, Any], int]:
+        data = {
+            TEST_ACC5: [
+                NFT(
+                    token_identifier='_nft_0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85_73552724610198397480670284492690114609730214421511097849210414928326607694469',  # noqa: E501
+                    background_color=None,
+                    image_url='https://openseauserdata.com/files/8fd18b22e4c81aff3998956e7a712d93.svg',  # noqa: E501
+                    name='nebolax.eth',
```

Comment:
this is duplicated in `test_lp_nfts_filtering`. Maybe you can move the nfts information to a variable in the file and then reuse this information?

Reply:
Good idea. Made it reusable